openshift_performance/ci/scripts/bad_actor/bad_actor.sh

```diff
@@ -0,0 +1,197 @@
+################################################
+## Auth=lhorsley@redhat.com, ematysek@redhat.com
+## Description: Cluster resilience testing via creation and deletion of a large number of projects (in parallel) on a cluster (OVN or SDN)
+## Polarion test case: OCP-41643 - Load cluster to test bad actor resilience	
+## https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-41643
+## Bug related: https://issues.redhat.com/browse/OCPBUGS-12266
+## Cluster config: 3 master (m5.2xlarge or equivalent) with 40 workers (The machine running the test should have at least 4 cores).
+##
+## Note: While the test runs, check the functionality of the cluster. A simple example (found in the Polarion test case):
+##       while true; do oc get co --no-headers| grep -v 'True.*False.*False'; oc get nodes --no-headers| grep -v ' Ready'; date; sleep 10; done
+################################################ 
+
+base_namespace="bad-actor"
+num_projects=2500
```

Comment:
In Eric's original script he had command line parameters to set some of these varaibles. It might be nice to have the same functionality 

#!/bin/bash

BASENAME=${1:-baa}
NUM_PROJECTS=${2:-1000}
num_procs=${3:-10}
sleep_mins=${4:-5}

https://raw.githubusercontent.com/RH-ematysek/svt/4968d67f253cfdd4b948df2b6faf3d965f0f033e/openshift_scalability/run_ba.sh

Reply:
Yeah, better to keep this functionality